app/views/spree/shared/_error_messages.html.haml

```diff
@@ -1,11 +1,10 @@
 - if target && target.errors.any?
   #errorExplanation.errorExplanation
     %h2
-      = Spree.t(:errors_prohibited_this_record_from_being_saved, count: target.errors.count)
-      \:
+      =  t("errors_prohibited_this_record_from_being_saved", count: target.errors.count)
```

Comment:
I agree with @luisramos0 's comment:

> I think we can use lazy lookup here.
> t(".errors_prohibited_this_record_from_being_saved")

Docs: https://guides.rubyonrails.org/i18n.html#lazy-lookup

Reply:
@rmklaus12 watch out for the validity of your translations tests (I never use the expected translation because it can be mixed up with translations coming from spree which is what is happening in your test, I always add a custom number to the end of the string), in this case, your call to t is not using lazy loading but your keys are correctly placed for the lazyloading. You need a dot here for your new translations to be picked up.

Reply:
Ah yes, thank you. Tested with extra characters and working with the dot in place now.